src/clusterfuzz/_internal/tests/core/bot/fuzzers/centipede/test_data/test_fuzzer.cc

```diff
@@ -0,0 +1,47 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
```

Comment:
Is this file copied from somewhere? Maybe write that in a comment if so.

Reply:
This file and the other one are copied from `honggfuzz`'s [`test_data/`](https://github.com/google/clusterfuzz/tree/c5d3476ef597addbfc8848b2c0ae1be1e428a55c/src/clusterfuzz/_internal/tests/core/bot/fuzzers/honggfuzz/test_data), and also exist in `libfuzzer`'s [`data/`](https://github.com/google/clusterfuzz/tree/c5d3476ef597addbfc8848b2c0ae1be1e428a55c/src/clusterfuzz/_internal/tests/core/bot/fuzzers/libFuzzer/data).